shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -34,11 +36,39 @@ impl ResultValue {
     }
 }
 
-fn assert_query_result(session: &Session, query: Statement, expected_rows: &[&[ResultValue]]) {
-    let result = session.execute(&query).wait().unwrap();
-    let result_rows: Vec<Vec<ResultValue>> = result
+/// Execute a `query` against the `session` and return result rows
+fn execute_query(session: &Session, query: &str) -> Vec<Vec<ResultValue>> {
+    let statement = stmt!(query);
+    let result = session.execute(&statement).wait().unwrap();
+    result
         .into_iter()
         .map(|x| x.into_iter().map(ResultValue::new).collect())
-        .collect();
-    assert_eq!(result_rows, expected_rows);
+        .collect()
+}
+
+/// Execute a `query` against the `session` and assert that the result rows match `expected_rows`
+fn assert_query_result(session: &Session, query: &str, expected_rows: &[&[ResultValue]]) {
+    let result_rows = execute_query(session, query);
+    assert!(expected_rows.len() == result_rows.len());
+    expected_rows
+        .iter()
+        .map(|x| x.to_vec())
+        .collect::<Vec<Vec<ResultValue>>>()
+        .iter()
+        .for_each(|item| {
+            if !result_rows.contains(item) {
+                panic!("{:?} missing from results", item);
+            }
+        });
+}
```

Comment:
This is a simpler way to express the same thing:
```rust
/// Execute a `query` against the `session` and assert that the result rows match `expected_rows`
fn assert_query_result(session: &Session, query: &str, expected_rows: &[&[ResultValue]]) {
    let result_rows = execute_query(session, query);
    assert!(expected_rows.len() == result_rows.len());
    for item in expected_rows {
        if !result_rows.contains(&item.to_vec()) {
            panic!("{:?} missing from results", item);
        }
    }
}
```

But I think we can do better:
```rust
// first turn ResultValue into derive soup
#[derive(Debug, Clone, PartialOrd, Ord, PartialEq, Eq)]
enum ResultValue {
    Text(String),
    Varchar(String),
    Int(i64),
    Boolean(bool),
}

/// Execute a `query` against the `session` and assert that the result rows match `expected_rows`
fn assert_query_result(session: &Session, query: &str, expected_rows: &[&[ResultValue]]) {
    let mut result_rows = execute_query(session, query);
    result_rows.sort();

    let mut expected_rows: Vec<_> = expected_rows.iter().map(|x| x.to_vec()).collect();
    expected_rows.sort();

    assert_eq!(result_rows, expected_rows);
}
```

This is a little simpler but most importantly provides a better error message.